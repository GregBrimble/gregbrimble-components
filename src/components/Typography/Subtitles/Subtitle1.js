import React from 'react';

import styled from 'styled-components';
import {Subtitle} from './shared';

const Subtitle1 = styled(Subtitle)`
    font-size: 1rem;
    line-height: 1.75rem;
`;

export default Subtitle1;
