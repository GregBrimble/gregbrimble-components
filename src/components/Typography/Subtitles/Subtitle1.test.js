import React from "react"
import Subtitle1 from "./Subtitle1"
import renderer from "react-test-renderer"

describe("Subtitle1", () => {
  it("renders properly", () => {
    const tree = renderer.create(<Subtitle1>Subtitle 1</Subtitle1>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
