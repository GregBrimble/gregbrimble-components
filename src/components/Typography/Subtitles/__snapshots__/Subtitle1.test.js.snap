// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Subtitle1 renders properly 1`] = `
<h6
  className="sc-bdVaJa sc-bwzfXH dVRZly"
>
  Subtitle 1
</h6>
`;
